homework/index.js

```diff
@@ -44,10 +70,14 @@
         return;
       }
       const ul = createAndAppend('ul', root);
-      repos.forEach(repo => renderRepoDetails(repo, ul));
+      const li = createAndAppend('li', ul);
+
+      repos
+  .sort((a, b) => a.name.localeCompare(b.name))
+  .forEach(repo => renderRepoDetails(repo, li));
```

Comment:
You are using a single list item that contains all the repositories. I would expect one list item per repository. So each `table` element should be wrapped in an `li`.

Reply:
Yes I reviewed another project for find table solution. I know this is not good for me. If you want  I  work to find another solution

Reply:
No, you don't need to find another solution as far as I'm concerned. But for a next time, please try for yourself first and ask help if you get stuck.

Reply:
yes you right I need to do like your said. Next time I will use only my solution. Thank you for this good feedback